engine/src/main/java/org/camunda/bpm/engine/impl/cfg/StandaloneProcessEngineConfiguration.java

```diff
@@ -38,7 +39,13 @@
     defaultCommandInterceptorsTxRequired.add(new LogInterceptor());
     defaultCommandInterceptorsTxRequired.add(new CommandCounterInterceptor(this));
     defaultCommandInterceptorsTxRequired.add(new ProcessApplicationContextInterceptor(this));
+
+    if (DbSqlSessionFactory.CRDB.equals(databaseType)) {
+      defaultCommandInterceptorsTxRequired.add(getCrdbRetryInterceptor());
+    }
```

Comment:
Let's move the retry interceptor to be the first one in the chain, so that we get logging once per command retry. Same applies to the other engine configurations.

Reply:
Refactored and mirrored in the Spring and JTA configurations.